diagnostic.js

```diff
@@ -32,7 +47,35 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+db.houses.update(
+   { House: 'Stark' },
+   {
+     members: ['Ned Stark', 'Arya Stark', 'Sansa Stark']
+   }
+)
+
+db.students.update(
```

Comment:
Two things here: You don't have the mongo id to work with here, and if you did, it would be a much longer hexadecimal code. Also, students?